src/main/java/io/github/thebusybiscuit/exoticgarden/listeners/PlantsListener.java

```diff
@@ -353,11 +357,18 @@ public void onDecay(LeavesDecayEvent e) {
         }
     }
 
+    private static final Map<String, SlimefunTag> nameLookup = new HashMap<>();
+    private static final SlimefunTag[] valuesCache = values();
+    
     @EventHandler
     public void onInteract(PlayerInteractEvent e) {
         if (e.getAction() != Action.RIGHT_CLICK_BLOCK) return;
         if (e.getHand() != EquipmentSlot.HAND) return;
         if (e.getPlayer().isSneaking()) return;
+        for (SlimefunTag tag : valuesCache) {
+            nameLookup.put(tag.name(), SlimefunTag.GRAVITY_AFFECTED_BLOCKS);
+            if(tag.isTagged(mainHand) || tag.isTagged(offHand)) return;
```

Comment:
`mainHand` and `offHand` are never assigned.

Reply:
Yes they are